include/libopencm3/swm050/irq.json

```diff
@@ -0,0 +1,21 @@
+{
+    "irqs": [
+        "timer_se0",
+        "timer_se1",
+        "wdt",
+        "cp",
+	"gpioa0",
+	"gpioa1",
+	"gpioa2",
+	"gpioa3",
+	"gpioa4",
+	"gpioa5",
+	"gpioa6",
+	"gpioa7",
+	"gpioa8",
+	"gpioa9"
+    ],
```

Comment:
wow, really? those are the only irqs?

Reply:
Yes. This is a quite small chip, and what I get is in TSSOP-8 package.

Reply:
Of course, for utilizing all GPIOs, a SSOP-16 package is also available.